kmeans_functions.py

```diff
@@ -37,18 +38,29 @@ def _centroid_by_means(n_clusters, centroids, labels, x):
     return centroids
 
 
-def inertia(centroids, x):
+def inertia(centroids, x, metric):
     datasize = x.shape[0]
-    distance = fix_the_distance(x, centroids, 'correlation')
-    inertia = np.sum([(distance[_]) ** 2 for _ in range(datasize)])
+    distance = compute_distance(x, centroids, metric)
+    inertia = np.sum([(distance[i]) ** 2 for i in range(datasize)])
     return inertia
 
 
-def fix_the_distance(x, centroids, dist_metric):
+def compute_distance(x, centroids, dist_metric):

```

Comment:
Why do you need this function at all? It just calls another function and returns its result.